JitsiTrackEvents.js

```diff
@@ -43,3 +43,18 @@ export const NO_DATA_FROM_SOURCE = 'track.no_data_from_source';
  * the microphone that is currently selected.
  */
 export const NO_AUDIO_INPUT = 'track.no_audio_input';
+
+/**
+ * Event fired when we detect local problem with the video track.
+ * First argument is the sourceName of the track and
+ * the seconds is a string indicating if the connection is currently
+ * - active - the connection is active
+ * - inactive - the connection is inactive, was intentionally interrupted by
+ * the bridge
```

Comment:
You might want to mention that the media was stopped by the bridge for this ep because of low BWE.